cmd/create/cluster/cmd.go

```diff
@@ -404,10 +412,31 @@ func init() {
 	)
 	flags.MarkHidden("properties")
 
+	flags.StringVar(
+		&args.mode,
+		"mode",
+		"",
+		"How to perform the operation. Valid options are:\n"+
+			"auto: Roles will be created using the current AWS account\n"+
+			"manual: Role files will be saved in the current directory",
+	)
+	Cmd.RegisterFlagCompletionFunc("mode", modeCompletion)
+
+	flags.StringVar(
+		&args.operatorRolesPermissionsBoundary,
+		"operator-roles-permissions-boundary",
```

Comment:
I think this should be `permissions-boundary` to match exactly the one in the `create operator-roles` command (since that's what the docs refer to)